### PR TITLE
JUnit5 tests of class Assurance - serve as example for other future t…

### PR DESCRIPTION
**Pull request by cergina**
…ests

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -80,7 +80,12 @@
             <artifactId>activation</artifactId>
             <version>1.1</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
```
Comment (xcipelova) [inline pom.xml:85]: junit je fajn

File: src/main/java/Model/Database/Support/Assurance.java [hunk 2]
```diff
@@ -14,37 +14,28 @@
...
-        return true;
+        return (value > 0);
     }
 
     public static boolean isVarcharOk(String value) {
-        if (null == value)
-            return false;
-
-        return true;
+        return (null != value);
```
Comment (xcipelova) [inline src/main/java/Model/Database/Support/Assurance.java:29]: super

File: src/test/java/Model/Database/Support/AssuranceTest.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+    @Test
+    void isDateOk() {
+        Date wannabeDate = Date.valueOf(LocalDate.now());
+
+        assertTrue(Assurance.isDateOk(wannabeDate));
+
+        wannabeDate = Date.valueOf(LocalDate.now().minusDays(30));
+
+        assertTrue(Assurance.isDateOk(wannabeDate));
```
Comment (xcipelova) [inline src/test/java/Model/Database/Support/AssuranceTest.java:24]: are you Ok?

**Review by xcipelova — COMMENTED** [review]
Pekne

File: src/test/java/Model/Database/Support/AssuranceTest.java (hunk 3 above)
Comment (cergina) [reply src/test/java/Model/Database/Support/AssuranceTest.java:24]: Very ok :D

**Review by ludko — APPROVED** [review]

**Pull request merged**